pkg/tsdb/postgres/postgres.go

```diff
@@ -48,20 +48,20 @@ func newPostgresQueryEndpoint(datasource *models.DataSource) (tsdb.TsdbQueryEndp
 }
 
 func generateConnectionString(datasource *models.DataSource, logger log.Logger) (string, error) {
-	sslmode := datasource.JsonData.Get("sslmode").MustString("verify-full")
+	sslmode := datasource.JsonData.Get("sslMode").MustString("verify-full")
```

Comment:
Changing the name of `sslmode` to `sslMode` is a breaking change. We could add a fallback here checking if `sslmode` is present, but same fallback is then needed client side.

Reply:
@marefr Good catch. Was there actually a reason for changing this setting name?